WolvenKit.Render/WolvenKit.Render.csproj

```diff
@@ -86,7 +86,7 @@
     <Reference Include="FSharp.Core, Version=4.6.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
       <HintPath>..\packages\FSharp.Core.4.6.2\lib\net45\FSharp.Core.dll</HintPath>
     </Reference>
-    <Reference Include="IrrlichtLime, Version=1.6.0.0, Culture=neutral, processorArchitecture=x86">
+    <Reference Include="IrrlichtLime, Version=1.6.1.0, Culture=neutral, processorArchitecture=AMD64">
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\Libs\IrrlichtLime.dll</HintPath>
```

Comment:
Can we have this as either a git submodule or simply add the solution to the project in the long run? It would be easier to debug it in the long run.

Reply:
Irrlicht.dll as well